fometask2/views.py

```diff
@@ -1,9 +1,21 @@
 from django.shortcuts import render
 from django.http import HttpResponse
 from .models import Client, Goods, Orders
+from django.utils import timezone
+from datetime import timedelta
 # Create your views here.
 
 
-def create_client(request):
-    help = "create new client"
+def sorted_orders(request, client_id):
 
+    days = 30
+    days = 2
+    days = 365
+
+    client = Client.objects.get(pk=client_id)
+    last_n_days = timezone.now() - timedelta(days=days)
+
+    recent_orders = Goods.objects.filter(orders__date__gte=last_n_days, orders__client=client)
```

Comment:
Вы не исключаете повторения.
```suggestion
    recent_orders = Goods.objects.filter(orders__date__gte=last_n_days, orders__client=client).distinct()
```